index.html

```diff
@@ -1,112 +1,157 @@
 <!DOCTYPE html>
 <html>
-<head>
-  <meta charset="UTF-8">
-  <title>Gridle</title>
-  <link rel="stylesheet" href="styles.css">
-</head>
-<body>
-
-
-<div class="container">
-  <div class="item1">
-    <div id="left-bar">
-            <button>
-                <img src="images/main-menu.svg" alt="menu" class="bar-icon">
-            </button>
-            <a href="https://www.youtube.com">
+  <head>
+    <meta charset="UTF-8" />
+    <title>YouTube Clone</title>
+    <link rel="stylesheet" href="styles.css" />
+  </head>
+  <body>
+    <div class="container">
+      <div class="item1">

```

Comment:
It's not a mistake to name the classes to your liking, but sometimes more descriptive names can make the structure of your application clearer.